open-metadata-implementation/adapters/open-connectors/data-store-connectors/cassandra-data-store-connector/src/main/java/org/odpi/openmetadata/adapters/connectors/datastore/cassandra/CassandraDataStoreConnector.java

```diff
@@ -0,0 +1,157 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+
+package org.odpi.openmetadata.adapters.connectors.datastore.cassandra;
+
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.CqlSessionBuilder;
+import com.datastax.oss.driver.api.core.metadata.schema.SchemaChangeListener;
+import org.odpi.openmetadata.adapters.connectors.datastore.cassandra.ffdc.CassandraDataStoreAuditCode;
+import org.odpi.openmetadata.frameworks.connectors.ConnectorBase;
+import org.odpi.openmetadata.frameworks.connectors.ffdc.ConnectorCheckedException;
+import org.odpi.openmetadata.frameworks.connectors.properties.ConnectionProperties;
+import org.odpi.openmetadata.frameworks.connectors.properties.EndpointProperties;
+import org.odpi.openmetadata.repositoryservices.auditlog.OMRSAuditLog;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.InetSocketAddress;
+
+/**
+ * The Cassandra data store connector provides the OCF connection objects for Apache Cassandra Database.
+ */
+public class CassandraDataStoreConnector extends ConnectorBase {
+    private static final Logger log = LoggerFactory.getLogger(CassandraDataStoreConnector.class);
+    private CassandraDataStoreAuditCode auditLog;
+    private OMRSAuditLog omrsAuditLog;
+
+    private CqlSession cqlSession;
+    private String username = null;
+    private String password = null;
+    private String serverAddresses = null;
+    private Integer port = 9042;
```

Comment:
This should be configurable, it should come from endpoint. It can be done in a separate PR.

Reply:
yes it can be also replaced from connectionProperties during the initialization.